server-client.c

```diff
@@ -914,7 +914,7 @@ server_client_check_redraw(struct client *c)
 		else if (c->prompt_string != NULL)
 			redraw = status_prompt_redraw(c);
 		else{
-			aux_status_redraw(c);
+			redraw = aux_status_redraw(c);
```

Comment:
How could this fix the issue? That variable is always overwritten immediately on the next line, right?